src/app/downloads/downloads-table.component.ts

```diff
@@ -41,7 +41,6 @@ export class DownloadsTableComponent implements OnDestroy {
     this.routerSub = this.store.select(selectRouter).subscribe((newRouterState: RouterModel) => {
       if (newRouterState) {
         if (isPodcastChanged(newRouterState, this.routerState)) {
-          this.store.dispatch(new ACTIONS.CastlePodcastAllTimeMetricsLoadAction());
```

Comment:
Is this not needed anymore?

Reply:
Nope, only needs to load podcast performance metrics in the castle effect when the selectSelectedPodcast selector fires. (Yeah, I'm seeing why ol' Todd prefixes those selector names with get instead of select.)